pkg/noderesourcetopology/filter.go

```diff
@@ -71,7 +72,8 @@ func singleNUMAContainerLevelHandler(pod *v1.Pod, zones topologyv1alpha2.ZoneLis
 		numaID, match := resourcesAvailableInAnyNUMANodes(logID, nodes, container.Resources.Requests, qos, nodeInfo)
 		if !match {
 			// we can't align container, so definitely we can't align a pod
-			return framework.NewStatus(framework.Unschedulable, fmt.Sprintf("cannot align container: %s", container.Name))
+			klog.V(2).InfoS("cannot align container", "name", container.Name, "kind", "app")
+			return framework.NewStatus(framework.Unschedulable, "cannot align app container")
```

Comment:
One thing that is bugging me is the `app container` I think just `container` is more accurate?

Reply:
that's no bother: I'll update to remove "app" so it will read
```
"cannot align container"
```

Reply:
👍